autonomous_software_pkg/src/target_generator.py

```diff
@@ -75,6 +77,10 @@ def __init__(self):
 
         """ Parameters """
         self.lookahead_distance = rospy.get_param("~lookahead_distance", 5)
+        self.velocity_mode = rospy.get_param(
+            "~velocity_mode", "WAYPOINT_FOLLOWING"
+        )  # "MANUAL_INPUT_SPEED" / "WAYPOINT_FOLLOWING"
+        self.speed_gain = rospy.get_param("~speed_gain", 1)
```

Comment:
Proposal: calling it `speed_scale_factor` or something like this, so that we don't risk confusing it with the future proportional term of the speed controller later on.

Reply:
Good point. Done